### PR TITLE
Readme Updates

### PR DESCRIPTION
**Pull request by ankitasharma88990**
there were very fewer grammatical mistakes they were corrected.

### CODE REVIEW

**Comment by ankitasharma88990** [comment]
please accept as it's the end of hacktoberfest.
Thank you.

File: readme.md [hunk 1]
```diff
@@ -29,7 +29,7 @@
 <br />
 
 * **Simple**: prompts has [no big dependencies](http://npm.anvaka.com/#/view/2d/prompts) nor is it broken into a [dozen](http://npm.anvaka.com/#/view/2d/inquirer) tiny modules that only work well together.
-* **User friendly**: prompt uses layout and colors to create beautiful cli interfaces.
+* ** User-friendly**: prompt uses layout and colours to create beautiful client interfaces.
```
Comment (koppor) [inline readme.md]: Why did you add a space here? Please be careful when updating text.

I also think, that `CLI` is the right term. Please do not mix up [CLI](https://en.wikipedia.org/wiki/Command-line_interface) with [client](https://en.wikipedia.org/wiki/Client_(computing)).

File: readme.md [hunk 2]
```diff
@@ -223,7 +223,7 @@ On cancel responses collected so far are returned.
 
 Type: `Function`
 
-Preanswer questions by passing an object with answers to `prompts.override`.
+Answer questions by passing an object with answers to `prompts.override`.
```
Comment (koppor) [inline readme.md]: This is not what the author intended. Please think of the program itself... These are pre-filled answers and not the answers by the user.

File: readme.md [hunk 3]
```diff
@@ -415,7 +415,7 @@ Type: `Function`
 
 Callback for when the state of the current prompt changes.
 The function signature is `(state)` where `state` is an object with a snapshot of the current state.
-The state object have two properties `value` and `aborted`. E.g `{ value: 'This is ', aborted: false }`
+The state object has two properties `value` and `aborted`. E.g `{ value: 'This is ', aborted: false }`
```
Comment (koppor) [inline readme.md]: Please add the missing dot after `E.g`!

File: readme.md [hunk 4]
```diff
@@ -660,15 +660,15 @@ Use <kbd>up</kbd>/<kbd>down</kbd> to navigate. Use <kbd>tab</kbd> to cycle the l
 | warn | `string` | Message to display when selecting a disabled option |
 | choices | `Array` | Array of strings or choices objects `[{ title, description, value, disabled }, ...]`. The choice's index in the array will be used as its value if it is not specified. |
 | onRender | `function` | On render callback. Keyword `this` refers to the current prompt |
-| onState | `function` | On state change callback. Function signature is an `object` with two properties: `value` and `aborted` |
+| on state | `function` | On state change callback. Function signature is an `object` with two properties: `value` and `aborted` |
```
Comment (koppor) [inline readme.md]: The program requires `onState`, so this change is wrong.

More reading on programming: https://teamtreehouse.com/community/variable-name-with-space-is-it-valid

Please take hacktoberfest as opportunity to level up your coding skills - and not as event to win a t-shirt.

File: readme.md [hunk 5]
```diff
@@ -714,7 +714,7 @@ The prompt will list options based on user input. Type to filter the list.
 Use <kbd>⇧</kbd>/<kbd>⇩</kbd> to navigate. Use <kbd>tab</kbd> to cycle the result. Use <kbd>Page Up</kbd>/<kbd>Page Down</kbd> (on Mac: <kbd>fn</kbd> + <kbd>⇧</kbd> / <kbd>⇩</kbd>) to change page. Hit <kbd>enter</kbd> to select the highlighted item below the prompt.
 
 The default suggests function is sorting based on the `title` property of the choices.
-You can overwrite how choices are being filtered by passing your own suggest function.
+You can overwrite how choices are being filtered bypassing your own suggest function.
```
Comment (koppor) [inline readme.md]: "passing" is a fixed termin in computer sciense. Please read on at https://www.mathwarehouse.com/programming/passing-by-value-vs-by-reference-visual-explanation.php

so, this change is invalid.

File: readme.md (hunk 1 above)
Comment (DrunkenPoney) [reply readme.md]: In fact _CLI_ stands for _C̲ommand L̲ine I̲nterface_

Comment (terkelg) [reply readme.md]: Please revert this change

File: readme.md [hunk 6]
```diff
@@ -96,7 +96,7 @@ const prompts = require('prompts');
 
 ### Prompt Chain
 
-Prompt with a list of prompt objects. Returns object with response.
+Prompt with a list of prompt objects. Returns object with a response.
```
Comment (terkelg) [inline readme.md]: Please revert this change change – or maybe change `a response` to `responses`? What do you think @DrunkenPoney @koppor?

File: readme.md (hunk 2 above)
Comment (terkelg) [reply readme.md]: Thank you @koppor. Please revert this change.

File: readme.md (hunk 5 above)
Comment (terkelg) [reply readme.md]: Please revert change

File: readme.md (hunk 6 above)
Comment (DrunkenPoney) [reply readme.md]: The number of responses goes with the number of prompt objects specified. If more than one object is provided then there would be more than one responses. So I think I'd say either "_Returns **an** object with one or more responses_" or just "_Returns **an** object with the responses_".

**Comment by koppor** [comment]
@terkelg Could you add the label "invalid"? 😇
